example/20_grouped_conv_bwd_weight/grouped_conv_bwd_weight_xdl_bf16.cpp

```diff
@@ -39,33 +39,33 @@ using DeviceConvBwdWeightInstance =
         WeiElementOp,         // WeiElementwiseOperation
         OutElementOp,         // OutElementwiseOperation
         ConvBwdWeightDefault, // ConvolutionBackwardWeightSpecialization
-        256,                  // BlockSize
-        128,                  // MPerBlock
-        128,                  // NPerBlock
-        4,                    // K0PerBlock
+        64,                   // BlockSize
+        16,                   // MPerBlock
+        16,                   // NPerBlock
+        32,                   // K0PerBlock
         8,                    // K1
-        32,                   // MPerXdl
-        32,                   // NPerXdl
-        2,                    // MXdlPerWave
-        2,                    // NXdlPerWave
-        S<1, 4, 16, 4>,       // ABlockTransferThreadClusterLengths_K0_M_K1
-        S<0, 3, 1, 2>,        // ABlockTransferThreadClusterArrangeOrder
-        S<0, 2, 1, 3>,        // ABlockTransferSrcAccessOrder
-        2,                    // ABlockTransferSrcVectorDim
-        8,                    // ABlockTransferSrcScalarPerVector
-        2,                    // ABlockTransferDstScalarPerVector_K1
-        true,                 // ABlockLdsAddExtraM
-        S<1, 4, 16, 4>,       // BBlockTransferThreadClusterLengths_K0_N_K1
-        S<0, 3, 1, 2>,        // BBlockTransferThreadClusterArrangeOrder
-        S<0, 2, 1, 3>,        // BBlockTransferSrcAccessOrder
-        2,                    // BBlockTransferSrcVectorDim
-        8,                    // BBlockTransferSrcScalarPerVector
-        2,                    // BBlockTransferDstScalarPerVector_K1
-        true,                 // BBlockLdsAddExtraN
+        16,                   // MPerXdl
+        16,                   // NPerXdl
+        1,                    // MXdlPerWave
+        1,                    // NXdlPerWave
+        S<4, 16, 1>,          // ABlockTransferThreadClusterLengths_K0_M_K1
```

Comment:
Because`ABlockTransferThreadClusterLengths_AK0_M_AK1` and `Sequence<AK0Number, MPerBlock, AK1Number>` need to have the same size, this requirement ensures proper `thread group slice transfer`. An assertion is in place for the division (/) operator. The sizes of both sequences are obtained for verification.